README.md

```diff
@@ -128,3 +128,15 @@ generates a zip archive `golden-results` which can be extracted into `tests/`.
 ```sh
 unzip golden-results.zip -d tests/
 ```
+
+### Fonts
+
+Fonts are served locally and copied into this Git repo from Google Fonts.
+
+To update existing font woff2 and `@font-face` CSS rules from Google Fonts, run
+`npm run build:fonts` from the `lit-dev-content` package.
```

Comment:
Could this be rolled into `npm run build` within `lit-dev-content`?

Reply:
It could be, but I actually was thinking it's preferable to isolate font updates as their own changes. Similar to the benefits of a package lock (although to be fair, fonts are waaay less likely to break something than a dependency, but it is still possible for a font to have a bug).

I'm thinking of adding a GitHub action soon which will automatically bring in the latest package lock, API docs, and fonts -- and automatically send us PRs when they change.